src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -1,110 +0,0 @@
-<?php
-
-/**
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- */
-declare(strict_types=1);
-
-namespace EzSystems\EzPlatformAdminUi\Tab\Dashboard;
-
-use eZ\Publish\API\Repository\ContentService;
-use eZ\Publish\API\Repository\ContentTypeService;
-use eZ\Publish\API\Repository\UserService;
-use eZ\Publish\Core\Helper\TranslationHelper;
-use eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProviderInterface;
-use EzSystems\EzPlatformAdminUi\Specification\ContentIsUser;
-use EzSystems\EzPlatformAdminUi\Specification\UserExists;
-use Pagerfanta\Pagerfanta;
-
-class PagerContentToDataMapper
```

Comment:
Technically it's a break BC break. To avoid it the class should be deprecated in v2.5 and then dropped in 3.0.